spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/SpatialConvolution.scala

```diff
@@ -576,6 +643,114 @@ class SpatialConvolution(
     needQuantize = value
     this
   }
+
+  /**
+   * Todo:
```

Comment:
Todo should be all in upper case for highlighting ?

Reply:
it is not case sensitive
this line is a placeholder, i have a plan to continue finishing what left, and it will be removed soon.

Reply:
OK. it's not case sensitive and normally if there is work left to finish, Upper case is the convention

Reply:
great, if there is a convention, then lets follow it.